nextflow_schema.json

```diff
@@ -0,0 +1,128 @@
+                },
+                "seq_type": {
+                    "type": "string",
+                    "description": "Sequence type."
+                },
+                "refseq": {
+                    "type": "boolean",
+                    "description": "Fetch only reference sequences.",
+                    "default": false
+                }
+            }
+        },
+        "cluster_options": {
+            "title": "Clustering options",
+            "type": "object",
+            "properties": {
+                "cluster_tool": {
+                    "type": "string",
+                    "description": "Clustering tool.",
+                    "enum": ["genes"]
+                }
+            }
+        },
+        "align_options": {
+            "title": "Multiple sequence alignment (MSA) options",
+            "type": "object",
+            "properties": {
+                "align_tool": {
+                    "type": "string",
+                    "description": "MSA tool.",
+                    "enum": ["mafft", "clustalo", "muscle"]
+                },
+                "align_out_format": {
+                    "type": "string",
+                    "description": "Output MSA file format.",
+                    "enum": ["clustal", "emboss", "fasta", "maf", "msf", "phylip", "nexus", "stockholm"],
+                    "default": "fasta"
+                }
+            }
+        },
+        "inference_options": {
+            "title": "Phylogenetic inference options",
+            "type": "object",
+            "properties": {
+                "inference_tool": {
+                    "type": "string",
+                    "description": "Phylogenetic inference tool.",
+                    "enum": ["fasttree", "raxml"]
+                },
+                "inference_out_format": {
+                    "type": "string",
+                    "description": "Output phylogeny file format.",
+                    "enum": ["newick", "nexus", "nexml", "phyloxml", "cdao"],
+                    "default": "newick"
+                }
+            }
+        },
+        "generic_options": {
+            "title": "Generic options",
+            "type": "object",
+            "description": "Less common options for the pipeline, typically set in a config file.",
+            "properties": {
+                "help": {
+                    "type": ["string", "boolean"],
+                    "description": "Display help text.",
+                    "hidden": true
+                },
+                "input_file": {
+                    "type": "string",
+                    "format": "file-path",
+                    "description": "Input file path."
+                },
+                "input_format": {
+                    "type": "string",
+                    "description": "Input file format.",
+                    "enum": ["clustal", "embl", "emboss", "fasta", "genbank", "gb", "maf", "msf", "phylip", "nexus", "stockholm"]
```

Comment:
Same as align and inference output format.

Reply:
Bear in mind this parameter won't match the GUI as this needs to gather all the options for both align and inference tools (since we do not know which one will be selected as the first stage).

Reply:
In the GUI, the same options are provided por both modules, as the tuples conversion list is used for both purposes (just like outformat). So, why does MEvoLib distinguish between both?

Reply:
I'm not sure I fully understand your question, sorry. Input and output formats should not be exactly the same, as the workflow as it is currently cannot accept input tree files such as `newick`. In any case, you are correct in flagging that this behaviour will be mirrored in the GUI.